namespace java thrift.test

struct myBinaryStruct {
  1: binary a
}
